MozillaReality/FirefoxReality/app/src/main/res/values/strings.xml

```diff
@@ -662,12 +676,16 @@
 
     <!-- This string labels the button in the privacy option panel that open the dialog to add/remove
          pop-up blocked sites. -->
-    <string name="security_options_block_pup_up_settings">Advanced</string>
+    <string name="security_options_block_pup_up_settings_v2">Advanced</string>
```

Comment:
Why was the ID changed, when the string didn't? This asks every locale to retranslate the string.

Reply:
That's a mistake, I'll revert that key.